### PR TITLE
Ali adam feed api challenge

### PR DESCRIPTION
**Pull request by AliAdam**

### CODE REVIEW

**Review by mapostolakis — COMMENTED** [review]
Hi @AliAdam, you almost passed the challenge!

Please, check my comments and let me know once you're done so I can review it again. Thank you 🙂

**Comment by AliAdam** [comment]
@mapostolakis Thanks a lot for your feedback .
i finished them please check.

File: FeedAPIChallenge/Feed API/FeedImageMapper.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	private enum HttpStatus: Int {
+		case ok = 200
+	}
+	private struct Items: Decodable {
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed API/FeedImageMapper.swift:15]: There is a line break missing between the enum and struct.

**Review by mapostolakis — COMMENTED** [review]
Hi @AliAdam,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs.

I left only one comment. Feel free to push the changes.

Great job! ✅

File: FeedAPIChallenge/Feed API/FeedImageMapper.swift (hunk 1 above)
Comment (AliAdam) [reply FeedAPIChallenge/Feed API/FeedImageMapper.swift:16]: Done

**Comment by AliAdam** [comment]
thanks @mapostolakis

**Comment by mapostolakis** [comment]
Awesome, @AliAdam!

**Pull request closed without merging**